.travis.yml

```diff
@@ -0,0 +1,64 @@
+language: cpp
+sudo: required
+dist: trusty
+  
+git:
+  depth: 3
+  lfs_skip_smudge: true
+   
+matrix:
```

Comment:
Why not build with more compilers?

Reply:
These are the lowest supported compiler versions. We can easily add more later if we want.